client/state/guided-tours/contexts/has-selected-site-defaults-site-title.js

```diff
@@ -1,16 +0,0 @@
-/**
- * Internal dependencies
- */
-import { getSelectedSiteId } from 'calypso/state/ui/selectors';
-import { hasDefaultSiteTitle } from 'calypso/state/sites/selectors';
```

Comment:
This removes the last usage of the `hasDefaultSiteTitle` selector - we could also remove it in this PR.

Reply:
Removed in 9cc2410cd7